script/Ciel/Day 12/12_01_CIEL12_5.txt

```diff
@@ -26,27 +28,29 @@ I'll go downstairs and get some water.
 [sha:9221ba65b1bd4e3b126566b6f2931a954088e93e]{
 -- Page 225, Offset 31497. Mods: @k, @e.
 -- 食堂に。
-To the dining room.
+I head for the dining room.
 }
 [sha:9221ba65b1bd4e3b126566b6f2931a954088e93e]{
 -- Page 225, Offset 31498. Mods: @k, @e.
 -- 食堂に。
-To the dining room.
+I head for the dining room.
 }
 [sha:9221ba65b1bd4e3b126566b6f2931a954088e93e]{
 -- Page 225, Offset 31499. Mods: @k, @e.
 -- 食堂に。
-To the dining room.
+I head for the dining room.
 }
 [sha:c7a7429e398d9a92a6de65cad2c4abe876580f83]{
 -- Page 225, Offset 31500.
 -- 午前零時過ぎ。
-It's past midnight.
+It's after midnight.
 }
 [sha:b0838e62aba9a7462f00584bc1ef6da127117538]{
 -- Page 225, Offset 31501.
 -- 屋敷の廊下に人影はなく、誰ともすれ違わないし、誰の無防備な背中も見つけられない。
-There isn't a single soul in sight in the mansion's corridor. No one to pass by, and no helpless backs to be seen.
+There isn't another soul to be seen in the hallways of the mansion.
+ There's nobody passing by.
+ No defenseless backs to be found.
```

Comment:
Don't love the not a single soul change. Feels too un-contracted.

Reply:
Then it could just be
>There's not a...

I feel like this collocation is too strong to just ignore. "Another soul" in reference to other people being missing is a fairly unnatural way of expressing this.

Reply:
I feel like it's pretty common, but I'll chalk it up to UK-go